src/utils/generateFileEntriesForArchiveRecords.ts

```diff
@@ -0,0 +1,25 @@
+import { getOriginalFileForArchiveRecord } from './getOriginalFileForArchiveRecord';
```

Comment:
Unfortunately I couldn't track this as a "move" in git because too much of the content changed alongside it.

Nothing about the implementation changed, just the variable names / called function names.